pallets/lockdown-mode/src/lib.rs

```diff
@@ -0,0 +1,149 @@
+#![cfg_attr(not(feature = "std"), no_std)]
+
+/// Edit this file to define custom logic or remove it if it is not needed.
+/// Learn more about FRAME and the core library of Substrate FRAME pallets:
+/// <https://docs.substrate.io/reference/frame-pallets/>
+pub use pallet::*;
+
+#[cfg(test)]
+mod mock;
+#[cfg(test)]
+mod tests;
+
+#[cfg(feature = "runtime-benchmarks")]
+mod benchmarking;
+pub mod weights;
+pub use weights::*;
+
+pub const ACTIVATED: bool = true;
+pub const DEACTIVATED: bool = false;
+
+#[frame_support::pallet]
+pub mod pallet {
+	use super::*;
+	use cumulus_primitives_core::{
+		relay_chain::BlockNumber as RelayBlockNumber, DmpMessageHandler,
+	};
+	use frame_support::{
+		pallet_prelude::{ValueQuery, *},
+		traits::Contains,
+	};
+	use frame_system::pallet_prelude::*;
+	use sp_std::vec::Vec;
+	use xcm_primitives::PauseXcmExecution;
+	#[pallet::pallet]
+	#[pallet::generate_store(pub(super) trait Store)]
+	pub struct Pallet<T>(_);
+
+	#[derive(Default)]
+	#[pallet::genesis_config]
+	/// Genesis config for lockdown mode pallet
+	pub struct GenesisConfig {}
+
+	#[pallet::genesis_build]
+	impl<T: Config> GenesisBuild<T> for GenesisConfig {
+		fn build(&self) {
+			LockdownModeStatus::<T>::put(ACTIVATED);
+		}
+	}
+
+	#[pallet::config]
+	pub trait Config: frame_system::Config {
+		type RuntimeEvent: From<Event<Self>> + IsType<<Self as frame_system::Config>::RuntimeEvent>;
+		type LockdownModeOrigin: EnsureOrigin<Self::RuntimeOrigin>;
+		type FilteredCalls: Contains<Self::RuntimeCall>;
```

Comment:
I think this name is confusing, it is hard to understand is it whitelisting or blacklisting. how about `WhiteListedCalls`?

Reply:
I take that, i think the purpose is to  black list , so it would be better: `BlackListedCalls`